ir/value.cpp

```diff
@@ -187,6 +189,11 @@ StateValue Input::toSMT(State &s) const {
   if (attributes & NonNull)
     s.addPre(Pointer(s.getMemory(), val).isNonZero());
 
+  if (getType().isPtrType()) {
+    auto nocap = Pointer(s.getMemory(), val).is_nocapture();
+    s.addAxiom((attributes & NoCapture) ? nocap : !nocap);
```

Comment:
I think this should be folded in mkInput/mkUndefInput, as it's much more efficient to concatenate another bit than adding an axiom.